man/systemd.exec.xml

```diff
@@ -3224,13 +3224,17 @@ StandardInputData=SWNrIHNpdHplIGRhIHVuJyBlc3NlIEtsb3BzLAp1ZmYgZWVtYWwga2xvcHAncy
 
         <varlistentry>
           <term><varname>$SERVICE_RESULT</varname></term>
+          <term><varname>$MONITOR_SERVICE_RESULT</varname></term>
```

Comment:
Maybe ON_FAILURE_SERVICE_RESULT etc? It's a bit vague otherwise

Reply:
I think calling this MONITOR makes sense, after all we do it for both OnFailureXYZ and OnSuccess=